style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color:hotpink;
+}
```

Comment:
- [ ] As the last code reviewers said please 🙏🏽 make sure that there are no linter errors. Kindly correct them in your `style.css` file 👇🏽 
![image](https://user-images.githubusercontent.com/58642949/191657139-df9e4883-96db-4370-a183-4956c5f263bb.png)
![image](https://user-images.githubusercontent.com/58642949/191657198-d6bd35bd-e39e-4f4b-83d8-797fcbd957c5.png)